ripple/src/main/webapp/businesscarousels.js

```diff
@@ -48,6 +48,7 @@ function getAddressComponent(components, type) {
   return components.filter((component) => component.types.indexOf(type) === 0).map((item) => item.long_name).pop() || null;
 }
 
+
```

Comment:
nit: is new line needed here?

Reply:
Removed the line! I think this PR included the commit that I had in my last one. The major differences are in the latest commit, which includes the redirectToBusinessInfo function and dynamically loads the under 20 mins away carousel.

Reply:
The relevant commits for this one are: connects business cards to business info page... and adds time-based query to businessgallery.js... Didn't want to just combine with old PR b/c its a good amount of new code.